backend/api/Areas/Admin/Controllers/ParcelController.cs

```diff
@@ -183,73 +197,118 @@ public IActionResult AddParcels([FromBody] ParcelModel[] models)
         /// <param name="model">The parcel model.</param>
         /// <returns>The parcel updated.</returns>
         [HttpPut]
-        public IActionResult UpdateParcel([FromBody] ParcelModel model)
+        public IActionResult UpdateParcel ([FromBody] ParcelModel model)
         {
-            var entity = _dbContext.Parcels
-                .Include(p => p.Address)
-                .Include(p => p.Buildings)
-                .Include(p => p.Buildings).ThenInclude(b => b.Address)
-                .Include(p => p.Buildings).ThenInclude(b => b.Address.City)
-                .Include(p => p.Buildings).ThenInclude(b => b.Address.Province)
-                .Include(p => p.Buildings).ThenInclude(b => b.BuildingConstructionType)
-                .Include(p => p.Buildings).ThenInclude(b => b.BuildingPredominateUse)
-                .SingleOrDefault(p => p.Id == model.Id);
-
-            if (entity == null) return BadRequest("Item does not exist");
-            var userId = this.User.GetUserId();
-            var address = entity.Address?.ToString();
-
-            _mapper.Map(model, entity);
+            var entity = _pimsAdminService.Parcel.Get (model.Id);
```

Comment:
Did you test this with Postman after making this change?

Reply:
I haven’t yet tested concurrency... need to do tonight.  Keep forgetting.  I have tested the endpoints with postman though

Reply:
I have to still create a InitUser endpoint for first time authenticated guest users for updates to work correctly.  Will work on this tomorrow.

Reply:
Created a `UserController.Activate()` endpoint that we will call when a user logins and does not have an assigned role or agency.  This will create the user so that all subsequent requests for that user will link him to the records they create or update.